cnf-tests/testsuites/e2esuite/multinetworkpolicy/multinetworkpolicy_sriov.go

```diff
@@ -0,0 +1,819 @@
+
+			// Traffic within other namespaces is allowed
+			eventually30s(nsY_podA).Should(Reach(nsY_podB))
+			eventually30s(nsZ_podA).Should(Reach(nsZ_podB))
+
+			// Traffic between other namespaces is allowed
+			eventually30s(nsY_podA).Should(Reach(nsZ_podA))
+			eventually30s(nsZ_podB).Should(Reach(nsY_podC))
+		})
+
+		It("ALLOW traffic to nsX_podA only from nsX_podB", func() {
+
+			MakeMultiNetworkPolicy(TEST_NETWORK,
+				WithPodSelector(metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						"pod": "a",
+					},
+				}),
+				WithIngressRule(multinetpolicyv1.MultiNetworkPolicyIngressRule{
+					From: []multinetpolicyv1.MultiNetworkPolicyPeer{{
+						PodSelector: &metav1.LabelSelector{
+							MatchLabels: map[string]string{
+								"pod": "b",
+							},
+						},
+					}},
+				}),
+				CreateInNamespace(nsX),
+			)
+
+			// The subject of test case
+			eventually30s(nsX_podB).Should(Reach(nsX_podA))
+			eventually30s(nsX_podC).ShouldNot(Reach(nsX_podA))
+
+			// Traffic that should not be affected
+			eventually30s(nsX_podB).Should(Reach(nsX_podC))
+		})
+
+		It("ALLOW traffic to nsX_podA only from (namespace == nsY)", func() {
+
+			MakeMultiNetworkPolicy(TEST_NETWORK,
+				WithPodSelector(metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						"pod": "a",
+					},
+				}),
+				WithIngressRule(multinetpolicyv1.MultiNetworkPolicyIngressRule{
+					From: []multinetpolicyv1.MultiNetworkPolicyPeer{{
+						NamespaceSelector: &metav1.LabelSelector{
+							MatchLabels: map[string]string{
+								"kubernetes.io/metadata.name": nsY,
+							},
+						},
+					}},
+				}),
+				CreateInNamespace(nsX),
+			)
+
+			// Allowed
+			eventually30s(nsY_podA).Should(Reach(nsX_podA))
+			eventually30s(nsY_podB).Should(Reach(nsX_podA))
+			eventually30s(nsY_podC).Should(Reach(nsX_podA))
+
+			// Not allowed
+			eventually30s(nsZ_podA).ShouldNot(Reach(nsX_podA))
+			eventually30s(nsZ_podB).ShouldNot(Reach(nsX_podA))
+			eventually30s(nsZ_podC).ShouldNot(Reach(nsX_podA))
+
+			eventually30s(nsX_podB).ShouldNot(Reach(nsX_podA))
+			eventually30s(nsX_podC).ShouldNot(Reach(nsX_podA))
+		})
+
+		It("ALLOW traffic to nsX_podA only from (nsY/* OR */podB)", func() {
+
+			MakeMultiNetworkPolicy(TEST_NETWORK,
+				WithPodSelector(metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						"pod": "a",
+					},
+				}),
+				WithIngressRule(multinetpolicyv1.MultiNetworkPolicyIngressRule{
+					From: []multinetpolicyv1.MultiNetworkPolicyPeer{
+						{
+							NamespaceSelector: &metav1.LabelSelector{
+								MatchLabels: map[string]string{
+									"kubernetes.io/metadata.name": nsY,
+								},
+							},
+						},
+						{
+							PodSelector: &metav1.LabelSelector{
+								MatchLabels: map[string]string{
+									"pod": "b",
+								},
+							},
+						},
+					},
+				}),
+				CreateInNamespace(nsX),
+			)
+
+			// Allowed
+			eventually30s(nsY_podA).Should(Reach(nsX_podA))
```

Comment:
Is it TCP traffic?

Reply:
Yes, TCP is the default. going to add it to the `Reach(...)` documentation.

https://github.com/openshift-kni/cnf-features-deploy/pull/1113/files#diff-3acaa25466e9df291fc9d8e7e5b2183c6bf51db11c5f494ebd5759dc75c3eae8R87